grpclb/src/test/java/io/grpc/grpclb/GrpclbLoadBalancerTest.java

```diff
@@ -248,7 +247,7 @@ public void errorPicker() {
   public void roundRobinPicker() {
     GrpclbClientLoadRecorder loadRecorder = new GrpclbClientLoadRecorder(timeProvider);
     Subchannel subchannel = mock(Subchannel.class);
-    RoundRobinEntry r1 = new RoundRobinEntry(DropType.RATE_LIMITING, loadRecorder);
+    RoundRobinEntry r1 = new RoundRobinEntry(loadRecorder, "LBTOKEN0001");
```

Comment:
In `RoundRobinEntry` class, it's better to define two static methods to create new instances, because the constructor does not have a meaningful name, and I don't realize it's a drop entry here unless I check its definition.